.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+# What's in this PR?
+
+# Why 
+
+# Whats Next?
+
+# How has this been tested?
```

Comment:
Does this section tell the reviewers how to test the changes in the PR. If so, shouldn't this be **How to test**?

Reply:
added a helpful hint under the header